app/assets/javascripts/proxy_status.js

```diff
@@ -7,11 +7,11 @@ function setItemStatus(item, response) {
   if(response.success) {
     item.attr('title', __('Active'));
     item.addClass('text-success');
-    item.html(icon_text('ok-sign', "", "glyphicon"));
+    item.html(icon_text('ok', "", "pficon"));
```

Comment:
does this change also the about page? IMHO we should change these icons globally (host list, about page, remote execution, scap etc).